### PR TITLE
Revise the content of the iOS documentation to reflect version 3.5.2.

### PR DESCRIPTION
**Pull request by settermjd**
This PR: 

- makes a number of changes, primarily around the flow of the content, but also around updating the images to reflect the latest version at the time of writing

### CODE REVIEW

**Comment by settermjd** [comment]
@nasli @michaelstingl any feedback or a 👍?

File: user_manual/ios_app.rst [hunk 1]
```diff
@@ -30,29 +30,49 @@ You'll also find links and information at the ownCloud `installation page
...
+
+When you tap any file in the Files view it downloads to your iPhone or iPad, if
 it hasn't already been downloaded, and opens a preview. 
+If the file has been successfully downloaded, then it will have a tick next to
+its icon, as in the image above. 
+
+Sharing Files
+~~~~~~~~~~~~~
+
+The little arrow button on the bottom left has options for sharing the file via
```
Comment (nasli) [inline user_manual/ios_app.rst]: The little arrow button on the bottom left has options for sharing the file via email, instant messaging or other of your apps, saving or copying, and printing.

File: user_manual/ios_app.rst [hunk 2]
```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
...
 Sharing Files
+~~~~~~~~~~~~~
+
+The triangular Share button opens a share dialog, allowing you to start
+`Sharing Files`_. 
+
+Deleting Files
+~~~~~~~~~~~~~~
+
+The trashcan allows you to delete the file from your device, your ownCloud
```
Comment (nasli) [inline user_manual/ios_app.rst]: check space here: trashcan

File: user_manual/ios_app.rst [hunk 3]
```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
 Click the button to make the file available, or not available offline.
 
 Sharing Files
+~~~~~~~~~~~~~
+
+The triangular Share button opens a share dialog, allowing you to start
```
Comment (nasli) [inline user_manual/ios_app.rst]: allowing you to share by link or with other ownCloud users
This option is different than the sharing files section previously described

File: user_manual/ios_app.rst [hunk 4]
```diff
@@ -30,29 +30,49 @@ You'll also find links and information at the ownCloud `installation page
...
+.. image:: images/ios-file-squirrel.png
+   :alt: Squirrel image in the iOS app
+
+Downloading Files
+~~~~~~~~~~~~~~~~~
+
+.. image:: images/ios-file-downloaded-file.png
+   :alt: A file that has been downloaded successfully in ownCloud
+
+When you tap any file in the Files view it downloads to your iPhone or iPad, if
```
Comment (nasli) [inline user_manual/ios_app.rst]: Just a suggest: any file or folder

File: user_manual/ios_app.rst [hunk 5]
```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
 Click the button to make the file available, or not available offline.
```
Comment (nasli) [inline user_manual/ios_app.rst]: Maybe add an explanation of what means this concept: That will keep your selected file or folder synchronized if any change is made in other client. Your files will always be downloaded and updated.

File: user_manual/ios_app.rst [hunk 6]
```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
...
+.. image:: images/ios-instant-uploads-videos-location-services-notification.png
+   :alt: ownCloud needs Location Services to be enabled for Instant Upload Videos to work.
+
+Once you have enabled one or both of the Instant Upload options, a new folder, called InstantUpload, will appear in your Files view. 
+If you click on it, you will see all of the photos and videos which have been automatically uploaded from your iPhone to your ownCloud account, as in the image below.
+
+.. image:: images/ios-instant-upload-file-list.png
+   :alt: Photos and videos which were automatically uploaded to ownCloud by Instant Upload.
+
+Sharing Files
```
Comment (nasli) [inline user_manual/ios_app.rst]: Please review "Editing Text Files" section, has wrong images

**Comment by nasli** [comment]
We are about to release a new 3.5.3 version and some UI changes in settings view so I suggest to wait until then for any new screenshot.

**Comment by settermjd** [comment]
Thanks for the feedback. I'll address it all today.

File: user_manual/ios_app.rst (hunk 4 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 1 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 5 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 3 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 2 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 6 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst [hunk 7]
```diff
@@ -125,9 +191,17 @@ Settings
...
 
 The Settings button (bottom right of any screen) takes you to the Settings 
-screen. You can add and edit new accounts, set a login PIN (personal 
-identification number), and it has links to Help, Recommend to a Friend, Send 
-Feedback, and your app version number. Click the information button at the 
+screen. 
+Here, you can:
+
+- Add and edit new accounts
+- Set a login PIN (personal identification number)
```
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst [hunk 8]
```diff
@@ -125,9 +191,17 @@ Settings
...
+- Get help
+- Recommend the app to a Friend
+- Send Feedback
+- Find the application version number
+
+Click the information button at the
```
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

**Comment by nasli** [comment]
@settermjd Changes has been updated here https://github.com/owncloud/ios/pull/903

**Pull request merged**